README.md

```diff
@@ -2,37 +2,37 @@
 
 ## Overview
 
-This repo contains a starter project for a simple key/value database server. During the interview exercise, you will pair with another Software Engineer to extend the functionality of the server.
```

Comment:
Most of the changes here are to use a consistent term ("server") to reference the project.